### ISSUE TITLE
Py 3.10+ support

### ISSUE THREAD

**Issue by thebjorn** [pypi/stdlib-list#58]
There have been several suggestions to do something similar to:
```
if python-version >= 3.10:
    return sys.stdlib_module_names
else:
    return old-algorithm 
```
Please don't do this, as the first half of the branch only returns the roots of the tree, while the else branch returns nested modules as well.

Something more like a recursive version of:
```
import sys


def find_names(roots, depth=2):
    res = list(roots)
    for root in roots:
        try:
            m = __import__(root)
            res.extend([f"{root}.{attr}" for attr in dir(m)])
        except (ModuleNotFoundError, ImportError):
            print(f"Module {root} not found")

    return res


if __name__ == '__main__':
    print(find_names(sys.stdlib_module_names))

```
(you may want to filter out the `antigravity` module...)

**Comment by woodruffw** [comment]
Thanks for the suggestions!

I still need to think about the plan here, but I think we'll probably end up checking in the generated lists in the old style: part of the value of this library is in the fact that it can list module names for a _particular_ Python version and not just the current version.

Your nested module approach looks reasonable -- we'll probably do a codegen variant of that in CI!

**Comment by woodruffw** [comment]
I've merged the lists for 3.10+, so this will be closed with #64.
